File: node-sdk/quickstart.mjs
/**
 * RoutePlex Node.js SDK — Quickstart
 *
 * npm install @routeplex/node
 */

import { RoutePlex } from "@routeplex/node";

const client = new RoutePlex({ apiKey: process.env.ROUTEPLEX_API_KEY });

const response = await client.chat("What is JavaScript?");
console.log(response.output);
console.log(`Model: ${response.modelUsed}`);
console.log(`Cost: $${response.usage.costUsd.toFixed(6)}`);

File: python-sdk/quickstart.py
"""
RoutePlex Python SDK — Quickstart

pip install routeplex
"""

import os
from routeplex import RoutePlex

client = RoutePlex(api_key=os.environ["ROUTEPLEX_API_KEY"])

# One-liner
response = client.chat("What is Python?")
print(response.output)
print(f"Model: {response.model_used}")
print(f"Cost: ${response.usage.cost_usd:.6f}")
